### PR TITLE
Update README.md instructions with OpenCV troubleshooting

### PR DESCRIPTION
**Pull request by VanDavv**
While I was building the core examples, I stumbled upon an issue with a missing OpenCV installation. This PR adds small README.md updates that should help others having this issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,20 @@ cd build
...
 
+## Running examples
+
+To run the tests build the library with the following options
+```
+cmake -H. -Bbuild -D DEPTHAI_BUILD_EXAMPLES=ON
+cmake --build build
+```
+
+Then navigate to `build/examples` folder and run a preffered test
```
Comment (szabi-luxonis) [inline README.md:142]: preferred example

File: README.md [hunk 2]
```diff
@@ -131,6 +131,20 @@ cd build
...
+```
+cd build
```
Comment (szabi-luxonis) [inline README.md:144]: cd build/examples

File: README.md [hunk 3]
```diff
@@ -131,6 +131,20 @@ cd build
...
+ctest
```
Comment (szabi-luxonis) [inline README.md:145]: ./{example_name}

running tests were described in previous paragraph

**Review by szabi-luxonis — CHANGES_REQUESTED** [review]

**Review by szabi-luxonis — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -131,6 +131,20 @@ cd build
...
+To run the tests build the library with the following options
+```
+cmake -H. -Bbuild -D DEPTHAI_BUILD_EXAMPLES=ON
+cmake --build build
+```
+
+Then navigate to `build/examples` folder and run a preferred example
+```
+cd build/examples
+./rgb_mobilenet
```
Comment (themarpe) [inline README.md:145]: Multi config generators (like Visual Studio on Windows) will have these in eg: `build/examples/[Debug/Release/...]/rgb_mobilenet`
Might be worth pointing this out

File: README.md [hunk 5]
```diff
@@ -173,6 +187,17 @@ This retains the libraries source code, so that debugger can step through it (th
 
 ## Troubleshooting
 
+### Build fails with missing OpenCV dependency
+
+If your build process happen to fail due to OpenCV library not being found, but you have the OpenCV installed, please
+run this command (replacing default Ubuntu path `` with yours)
+
+```
+export OpenCV_DIR=/usr/lib/x86_64-linux-gnu/cmake/opencv4
```
Comment (themarpe) [inline README.md:196]: Probably preferred to add this to `cmake` command when configuring with:
```
cmake -H. -Bbuild -D OpenCV_DIR=/usr/lib/x86_64-linux-gnu/cmake/opencv4
```

To keep this settings "local"

File: README.md [hunk 6]
```diff
@@ -173,6 +187,17 @@ This retains the libraries source code, so that debugger can step through it (th
 
 ## Troubleshooting
 
+### Build fails with missing OpenCV dependency
+
+If your build process happen to fail due to OpenCV library not being found, but you have the OpenCV installed, please
+run this command (replacing default Ubuntu path `` with yours)
```
Comment (themarpe) [inline README.md:193]: > run this command (replacing default Ubuntu path `` with yours)

Dual backticks there a typo?

**Review by themarpe — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -142,9 +142,12 @@ cmake --build build
 Then navigate to `build/examples` folder and run a preferred example
 ```
 cd build/examples
-./rgb_mobilenet
+./rgb_mobilenet 
 ```
 
+Please be aware that multi config generators (like Visual Studio on Windows) will have the examples built in `build/examples/Debug/Release/.../rgb_mobilenet`
```
Comment (themarpe) [inline README.md:148]: Please be aware that multi config generators (like Visual Studio on Windows) will have the examples built in `build/examples/[Debug/Release/...]/rgb_mobilenet`

**Review by themarpe — CHANGES_REQUESTED** [review]

**Review by themarpe — APPROVED** [review]

**Pull request merged**